oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -467,6 +467,6 @@ private long getProcessOpenFileLimit(long processId, int index) {
 
         // Split all non-Digits away -> ["", "{soft-limit}, "{hard-limit}"]
         final String[] split = maxOpenFilesLine.get().split("\\D+");
-        return Long.parseLong(split[index]);
+        return ParseUtil.parseLastLong(split[index], 0);
```

Comment:
Looks like we use -1 as the default in the previous lines.

Also use the ..`orDefault()` version.

Reply:
Done